package.json

```
@@ -3,10 +3,9 @@
   "bugs": "https://github.com/micromark/micromark-extension-mdx-expression/issues",
   "description": "",
   "devDependencies": {
-    "@types/acorn": "^4.0.0",
     "@types/estree": "^1.0.0",
     "@types/node": "^22.0.0",
-    "acorn": "^8.0.0",
+    "acorn": "^8.11.0",
```

We don’t need to narrow the semver range. Both ranges resolve to the same version.

```suggestion
    "acorn": "^8.0.0",
```